sensors_compose.yaml

```diff
@@ -0,0 +1,61 @@
+services:
+
+  av_car_description:
+    image: "ghcr.io/ipab-rad/av_car_description:1.0.1"
+    container_name: av_car_description
+    network_mode: "host"
+    privileged: true
+    volumes:
+      - "/dev/shm:/dev/shm"
+
+  av_dbw:
+    image: "ghcr.io/ipab-rad/av_dbw:1.0.0"
+    container_name: av_dbw
+    network_mode: "host"
+    privileged: true
+    volumes:
+      - "/dev/shm:/dev/shm"
+      - "/dev/bus/usb:/dev/bus/usb"
```

Comment:
Minor nitpick: Not sure which of these do/don't need usb volumes. If you have already tested this container without this on the vehicle feel free to resolve this conversation.

Reply:
I think the ROS1 dbw needs this mounting, otherwise `Dataspeed CAN Tool Not Found`  however, we could have been getting away by just doing `/dev:/dev`